.clang-tidy

```diff
@@ -25,11 +25,23 @@ Checks: >
   abseil-*,
   bugprone-*,
   -bugprone-macro-parentheses,
+  -bugprone-easily-swappable-parameters,
+  -bugprone-narrowing-conversions,
+  -bugprone-implicit-widening-of-multiplication-result,
+  cert-*,
+  cppcoreguidelines-*,
+  -cppcoreguidelines-non-private-member-variables-in-classes,
+  -cppcoreguidelines-narrowing-conversions,
+  -cppcoreguidelines-avoid-magic-numbers,
+  -cppcoreguidelines-avoid-do-while,
   google-*,
   -google-readability-todo,
```

Comment:
Curious on this as well

Reply:
this was subtracted out before idk why i feel like most todos have names, can add back